### PR TITLE
SAK-44746 GithubActions test exclusions

### PR DESCRIPTION
**Pull request by ern**
Add profile to setup test exclusions and add StatsUpdateManagerTests

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Build with Maven
       env:
         MAVEN_OPTS: -Djava.awt.headless=true -Dorg.slf4j.simpleLogger.defaultLogLevel=info -Dmaven.wagon.http.retryHandler.count=2 -Dmaven.wagon.http.pool=true
-      run: mvn test --batch-mode
+      run: mvn -PskipBrokenTestsOnUbuntu test --batch-mode
```
Comment (ottenhoff) [inline .github/workflows/maven.yml:34]: doesn't really matter what this profile is called but we dont' really think it's ubuntu related..... i can replicate locally with mysql on mac ......

Comment (ern) [reply .github/workflows/maven.yml:34]: its a different failing test on mac so its not the same, but as you say it doesnt matter....

Comment (ern) [inline .github/workflows/maven.yml:34]:
```suggestion
      run: mvn -PskipBrokenTests test --batch-mode
```

File: sitestats/sitestats-impl/pom.xml [hunk 2]
```diff
@@ -12,6 +12,24 @@
         <version>22-SNAPSHOT</version><!-- sitestats.version -->
     </parent>
 
+    <profiles>
+        <profile>
+            <id>skipBrokenTestsOnUbuntu</id>
```
Comment (ern) [inline sitestats/sitestats-impl/pom.xml:17]:
```suggestion
            <id>skipBrokenTests</id>
```

**Review by ottenhoff — APPROVED** [review]

**Pull request merged**